lib/main_common.pm

```diff
@@ -2729,7 +2729,12 @@ sub load_hypervisor_tests {
     }
 
     if (check_var('VIRT_PART', 'hotplugging')) {
-        loadtest 'virtualization/universal/hotplugging';    # Try to change properties of guests
```

Comment:
This hotplugging test is also used in developing product test, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/sle/main.pm#L637. Please also update it .